seacrowd/sea_datasets/cc_aligned_sent/cc_aligned_sent.py

```diff
@@ -0,0 +1,225 @@
+      Tang, Yuqing  and
+      Guzm{\'a}n, Francisco  and
+      Schwenk, Holger  and
+      Koehn, Philipp",
+    editor = "Bojar, Ond{\v{r}}ej  and
+      Chatterjee, Rajen  and
+      Federmann, Christian  and
+      Fishel, Mark  and
+      Graham, Yvette  and
+      Haddow, Barry  and
+      Huck, Matthias  and
+      Yepes, Antonio Jimeno  and
+      Koehn, Philipp  and
+      Martins, Andr{\'e}  and
+      Monz, Christof  and
+      Negri, Matteo  and
+      N{\'e}v{\'e}ol, Aur{\'e}lie  and
+      Neves, Mariana  and
+      Post, Matt  and
+      Turchi, Marco  and
+      Verspoor, Karin",
+    booktitle = "Proceedings of the Fourth Conference on Machine Translation (Volume 3: Shared Task Papers, Day 2)",
+    month = aug,
+    year = "2019",
+    address = "Florence, Italy",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/W19-5435",
+    doi = "10.18653/v1/W19-5435",
+    pages = "261--266",
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ind", "jav", "sun", "tha", "vie", "zlm", "lao", "khm", "mya", "ceb"]
+_DATASETNAME = "cc_aligned_sent"
+_DESCRIPTION = """\
+This dataset contains the sentence pairs extracted from CC-Aligned document
+pairs using similarity scores of LASER embeddings (minimum similarity 1.04,
+sorted based on decreasing similarity score). It misses some languages not
+covered by LASER.
+"""
+
+_HOMEPAGE = "https://www2.statmt.org/cc-aligned/"
+_LICENSE = Licenses.UNKNOWN.value
+_URLS = {
+    "ind": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-id_ID.tsv.xz",
+    "jav": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-jv_ID.tsv.xz",
+    "sun": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-su_ID.tsv.xz",
+    "tha": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-th_TH.tsv.xz",
+    "vie": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-vi_VN.tsv.xz",
+    "zlm": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-ms_MY.tsv.xz",
+    "lao": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-lo_LA.tsv.xz",
+    "khm": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-km_KH.tsv.xz",
+    "mya": "https://data.statmt.org/cc-aligned/sentence-aligned/en_XX-my_MM.tsv.xz",
+    "ceb": "https://data.statmt.org/cc-aligned/sentence-aligned/cx_PH-en_XX.tsv.xz",
+}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_SUBSETS = ["id_ID", "jv_ID", "su_ID", "th_TH", "vi_VN", "ms_MY", "lo_LA", "km_KH", "my_MM", "cx_PH"]
+
+
+class CCAlignedSentencesDataset(datasets.GeneratorBasedBuilder):
+    """CC Aligned Sentences dataset by Chaudhary et al., (2019)"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "t2t"
+
+    # Add configurations for loading a dataset per language.
+    dataset_names = sorted([f"{_DATASETNAME}_{lang}" for lang in _LANGUAGES])
```

Comment:
Imo, it's better if we use the `_SUBSET` instead of `_LANGUAGES` for config creations

Reply:
Got it! Should be in the same commit as well :)